### PR TITLE
Update language in README, etc

### PR DESCRIPTION
**Pull request by ds6**
I rewrote some of the text in README.md, CONTRIBUTING.md, lesspass-site/index.html, and also updated some links so they work again. Not part of this PR but I also updated the wiki.

What is the status of the website rewrite in #459? Is that a full rewrite or just the updates it has had so far? I noticed there were commits that updated it but these changes have not been pushed to the public site. In my quick research to find a stateless password manager I almost overlooked LessPass due to the broken links, as that signals to me the project is not worth my time; thankfully I looked into it and saw that it was a very nice project and decided to update some of the text.

I may also write some documentation on docker deployment for #470 because I will likely be starting my own instance soon. Some of the other technical wiki pages could also use updating.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,34 +2,45 @@
...
- * yarn
+Before diving in to the code, you'll need these tools installed in your environment:
+
+ * [NodeJS](https://nodejs.org/)
+ * [Yarn](https://yarnpkg.com/)
+
+The mobile apps need:
+
+ * [Gradle](https://developer.android.com/studio/build/) for Android
+ * [Xcode](https://developer.apple.com/xcode/) for iOS
```
Comment (guillaumevincent) [inline CONTRIBUTING.md]: There is no iOS app right now but yes in the futur will need this

**Comment by guillaumevincent** [comment]
> What is the status of the website rewrite in #459?

I need to find some time to update everything to the latest version everywhere.
I will have more time with the next couple of month

Thank you for the pull request

**Comment by ds6** [comment]
Oops, I accidentally [invoked the name of the demon](https://github.com/ds6/lesspass/commit/0cc7197f44b87f1feefb99316c47865f382a4870)... Does that warrant a new PR?

**Comment by guillaumevincent** [comment]
@ds6 I don't understand, the Lastpass name wasn't in the final pull request?

**Comment by ds6** [comment]
[Correct](https://github.com/lesspass/lesspass/blob/master/CONTRIBUTING.md).

**Comment by guillaumevincent** [comment]
oh miss this one, let me fix this

**Pull request merged**